website/pages/zh/index.json

```diff
@@ -22,10 +22,7 @@
       "title": "创建子图",
       "description": "在子图工作室中创建子图"
     },
-    "migrateFromHostedService": {
-      "title": "Upgrade from the hosted service",
-      "description": "Upgrading subgraphs to The Graph Network"
-    }
+    "migrateFromHostedService": {}
```

Comment:
Missing this one, but whatever. I believe it's gonna be removed automatically by Crowdin. Approving now :)